components/scream/tests/rrtmgp/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+INCLUDE (ScreamUtils)
+
+# Required libraries
+SET (NEED_LIBS rrtmgp scream_control scream_share)
+
+# Test atmosphere processes
+CreateUnitTest(rrtmgp_stand_alone "rrtmgp_stand_alone.cpp" "${NEED_LIBS}")
+
+# Copy RRTMGP absorption coefficient lookup tables to local data directory
+FILE (MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/data)
+CONFIGURE_FILE(${SCREAM_DATA_DIR}/rrtmgp-data-sw-g224-2018-12-04.nc
```

Comment:
We will need to include these two netCDF data files in the PR, otherwise the code won't be able to build

Reply:
They already exist in the RRTMGP source, would it be better to have cmake copy them from there? `components/cam/src/physics/rrtmgp/external/rrtmgp/data`

Reply:
My sense is that the answer is yes.  At least from the vantage that if those files are updated in Radiation we don't have to also make sure to copy them to somewhere else in scream.  @jgfouca may have an opinion though on the proper way to handle data input for processes.  In this case, radiation is a submodule so we may treat it differently than say P3, where we have a local copy of the input data.

Reply:
How big are those files?

Reply:
@bartgol 46MB

Reply:
That's not tiny. If it's already in rrtmgp repo, I would use that, rather than add that file to the scream repo too. Sounds like a waste of space.

Reply:
@bartgol agreed.